src/plugins/vis_type_timeseries/public/application/visualizations/views/timeseries/index.js

```diff
@@ -73,16 +73,16 @@ export const TimeSeries = ({
   const chartRef = useRef();
 
   useEffect(() => {
-    const updateCursor = (_, cursor) => {
-      if (chartRef.current) {
+    const updateCursor = ({ cursor }) => {
+      if (chartRef.current && cursor) {
```

Comment:
I'm not sure whether the check of `cursor` is needed here, could be omited

Reply:
done